torch/_refs/__init__.py

```diff
@@ -355,12 +358,31 @@ def _ref(a: TensorLikeType) -> TensorLikeType:
     type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT,
 )
 
+exp2 = _make_elementwise_unary_reference(
+    prims.exp2,
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT,
+    aten_op=torch.ops.aten.exp2,
```

Comment:
This shouldn't be necessary because its name is already `exp2`, right?

Reply:
Ah, right. Had added it while prototyping the decomposition. Will remove it. Thanks!